packages/sanity/src/core/field/diff/components/VersionInlineBadge.tsx

```diff
@@ -0,0 +1,17 @@
+import {type BadgeTone} from '@sanity/ui'
+import {css, styled} from 'styled-components'
+
+/**
+ * @internal
+ */
+export const VersionInlineBadge = styled.span<{$tone?: BadgeTone}>((props) => {
```

Comment:
Wonder if it makes sense to move this to a separate release folder. 
Mainly thinking that we kinda have something like this everywhere and might be good to eventually put it in a single place and use it everywhere instead of what we have now 🤔
Not necessarily for this PR as it is already massive, but a thought

Reply:
Agree, moved to `core/releases/components`